### PR TITLE
Fix Wasm logo and add JS, Erlang, Godot and Nix

### PR DESCRIPTION
**Pull request by DanielleMaywood**
The Wasm logo is off-centre, also adds a JS Lucy.

### CODE REVIEW

**Comment by DanielleMaywood** [comment]
Added Nix and Erlang

File: branding.html [hunk 1]
```diff
@@ -84,6 +84,11 @@ <h4>✨ How clumsy as Lucy?</h4>
   <li><img src="/images/lucy/lucygay.svg" alt="Lucy gay" /></li>
   <li><img src="/images/lucy/lucybi.svg" alt="Lucy bi" /></li>
   <li><img src="/images/lucy/lucywasm.svg" alt="Lucy Wasm" /></li>
+  <li><img src="/images/lucy/lucyjs.svg" alt="Lucy JS" /></li>
+  <li><img src="/images/lucy/lucyerl.svg" alt="Lucy Erlang" /></li>
+  <li><img src="/images/lucy/lucynix.svg" alt="Lucy Nix" /></li>
+  <li><img src="/images/lucy/lucyflake.svg" alt="Lucy Flake" /></li>
+  <li><img src="/images/lucy/lucygodot.svg" alt="Lucy Godot" /></li>
```
Comment (lpil) [inline branding.html:91]: It would be nice to credit you as the artist somehow. Alt text perhaps?

**Review by lpil — COMMENTED** [review]
These are the cutest things ever

File: branding.html (hunk 1 above)
Comment (DanielleMaywood) [reply branding.html:91]: I personally don't mind if I'm credited or not.

We should probably ask @isaacharrisholt if they want crediting for `lucyflake.svg` as all I did was convert their design into an svg.

Comment (isaacharrisholt) [reply branding.html:91]: I also don't mind, but if I do, it's only fair that @DanielleMaywood gets joint credit

Comment (lpil) [reply branding.html:91]: I want to record it! 😁

**Comment by isaacharrisholt** [comment]
Side note, I've just noticed a typo in the FAQ, where it says 'How clumsy **as** Lucy?'. Is it worth just adding that small fix to this PR, or should I create a new one?

**Comment by lpil** [comment]
Either or works for me!

**Review by lpil — APPROVED** [review]
💜 💜 💜

**Pull request merged**